tests/dragonfly/conftest.py

```diff
@@ -28,6 +28,30 @@
 logging.getLogger("asyncio").setLevel(logging.WARNING)
 
 DATABASE_INDEX = 0
+BASE_LOG_DIR = "/tmp/dragonfly_logs/"
+
+
+def pytest_configure(config):
```

Comment:
plz add a comment that this runs on pytest start -- I had to google this to figure out what this is doing